src/main/webapp/index.jsp

```diff
@@ -0,0 +1,175 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Place Search Pagination</title>
+
+    <script src="https://polyfill.io/v3/polyfill.min.js?features=default"></script>
+
+    <script
+      src="https://maps.googleapis.com/maps/api/js?key=XXXXXXX&callback=initMap&libraries=places&v=weekly"
+      defer
+    ></script>
+
+    <script src="https://polyfill.io/v3/polyfill.min.js?features=default"></script>
+
+    <style type="text/css">
```

Comment:
i'd recommend putting the css and js stuff in separate files (best practices lol)